lib/internal/process/main_thread_only.js

```diff
@@ -98,6 +104,18 @@ function setupPosixMethods(_initgroups, _setegid, _seteuid,
     }
   };
 
+  function validateInc(inc) {
+    const tInc = typeof inc;
+    if (tInc === 'number') {
+      validateInt32(inc, 'nice');
+      return _nice(inc);
+    } else if (tInc === 'undefined') {
+      return _nice(inc);
```

Comment:
I think it’s a bit confusing to mix validation and the actual calls to `_nice` … could you separate them?

Reply:
Split them up.:)